javascript/javascript1/week1/pizza.js

```diff
@@ -0,0 +1,24 @@
+console.log("I love pizza");
+const favPizza = "Margarita";
+const pizzaPrice = 100;
+console.log(`New Pizza Order: ${favPizza}`);
+console.log(`The Price of the Pizza is: ${pizzaPrice}`);
+
+// 2part of exersice
+
+const NumPizza = 3;
+let strFamily;
+let totalPrice;
+totalPrice = (pizzaPrice * NumPizza);
+
+if (strFamily = true) {
```

Comment:
This is assigning `true` to `strFamily`.
I think what you're looking for is either `==` or `===`.
See all the [conditional operators here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators#comparison_operators).